api/src/main/java/org/openmrs/module/Module.java

```diff
@@ -19,6 +19,7 @@
 import java.util.Properties;
 import java.util.Set;
 import java.util.Vector;
+import java.lang.*;
```

Comment:
Do not use wildcard imports.

Please see:
https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Imports